tests/unit/geo.tcl

```diff
@@ -535,4 +539,42 @@ start_server {tags {"geo"}} {
         set test_result
     } {OK}
     }
+
+    test {GEOSEARCH fuzzy test} {
+        if {$::accurate} { set attempt 300 } else { set attempt 30 }
+        while {[incr attempt -1]} {
+            unset -nocomplain debuginfo
+            set srand_seed [clock milliseconds]
+            lappend debuginfo "srand_seed is $srand_seed"
+            expr {srand($srand_seed)} ; # If you need a reproducible run
+            r del mypoints
+
+            geo_random_point search_lon search_lat
+            set width_m [expr {[randomInt 1000]+10}]
+            set height_m [expr {[randomInt 1000]+10}]
```

Comment:
maybe we should have some mechanism to randomly also choose very big boxes, or very long and narrow ones.

Reply:
The current code also satisfies the narrow and long condition. Although the current box length and width are limited to 1000 meters, the mobile box only moves 1 meter, so it can maintain a 0.1% error rate.

Although 1,000 meters is not large enough, it is long and narrow for 1 meter.